test/dialogmote/AvlysDialogmoteSkjemaTest.tsx

```diff
@@ -367,7 +367,7 @@ describe("AvlysDialogmoteSkjemaTest", () => {
       .flatMap((documentComponent) => documentComponent.texts)
       .forEach((text) => {
         expect(within(forhandsvisningAvlysningBehandler).getByText(text)).to
-          .exist;
+          .include;
```

Comment:
Er dette noe vi synes er greit? 🤔 Problemet er en tekst i avlysning av dialogmøte dokumentet som sier noe om minuttet det ble sendt på. Dette ble tilfeldigvis feil i testene i CI/CD på Github. 
Kan la det være også…

Reply:
Hva gjør `.include` her?

Reply:
Slik jeg har skjønt det sjekker det om teksten inkluderer det vi sjekker mot, og ikke nødvendigvis at det er eksakt match. Muligens dette også vil feile siden teksten vi sjekker mot ikke inkluderer "hele" dato og tidspunkt for utsending 🤔 
Kan la det være nå. Er jo egentlig ikke relatert til denne oppgaven